functional/FunctionalTest.java

```diff
@@ -407,7 +409,7 @@ public static void skipStream(InputStream stream, int len) throws Exception {
     }
   }
 
-  private static void handleException(String methodName, String args, long startTime, Exception e)
+  static void handleException(String methodName, String args, long startTime, Exception e)
```

Comment:
Please restore this ineffective diff.

Reply:
@balamurugana given the failing tests, I realized this was needed as we use this method in `TestMinioAdminClient.java`. Any opinions on being able to use this function there?

Reply:
Does `public` instead of `private` work for you?

Reply:
Yup!